core/domain/config_domain.py

```diff
@@ -334,10 +364,14 @@ def get_config_property(cls, name):
         Returns:
             instance. The instance of the specified configuration property.
         """
-        return cls._config_registry.get(name)
+        return cast(ConfigProperty, cls._config_registry.get(name))
```

Comment:
```suggestion
        return cls._config_registry[name]
```

Reply:
Done!

Reply:
doing this falling backend tests as well as lighthouse cl test

Reply:
What backend tests?

Reply:
i'm bit confused why this is happening!!
you can find failng tests here:
https://github.com/oppia/oppia/runs/4699748232?check_suite_focus=true#step:8:1835
https://github.com/oppia/oppia/runs/4699748232?check_suite_focus=true#step:8:1861